Code/CMakeLists.txt

```diff
@@ -252,7 +252,8 @@ if(SV_USE_QT_GUI)
     "Source/sv4gui/Modules/Mesh/Common"
     "Source/sv4gui/Modules/Simulation"
     "Source/sv4gui/Modules/ProjectManagement"
-    "Source/sv4gui/Modules/QtWidgets")
+    "Source/sv4gui/Modules/QtWidgets"
```

Comment:
Is there a reason the indents are different?

Reply:
@gmaher - why are the indents different?

Reply:
They should be the same now. Some lines had different indents in my editor so I went back and used spaces to indent them.